docs/user/dql/basics.rst

```diff
@@ -1099,3 +1099,20 @@ Result set:
 +--------------+
 
 
+Offset position can be given following the OFFSET keyword as well, here is an example::
+
+    >od SELECT age FROM accounts ORDER BY age LIMIT 2 OFFSET 1
+    fetched rows / total rows = 2/2
+    +-------+
+    | age   |
+    |-------|
+    | 32    |
+    | 33    |
+    +-------+
+
+
+Limitation
+----------
+Generally, sort plan is pushed down into the Elasticsearch DSL in plan optimization, but note that if a query has complex sorting, like sort expression, which would not be pushed down during optimization (see `Optimizations <../optimization/optimization.rst>`_ for details), but computed in local memory. However, the engine fetches the index of a default size that is set in plugin setting (See for `Optimizations <../admin/settings.rst#opendistro.query.size_limit>` details). Therefore, the result might not be absolutely correct if the index size is larger than the default size of index scan. For example, the engine has a index scan size of 200 and the index size is 500. Then a query with limit 300 can only fetch 200 rows of the index, compute and return the sorted result with 200 rows, while the rest 300 rows of the index are ignored and would not be fetched into the engine. To get an absolutely correct result, it is suggested to set the query size limit to a larger value before run the query.
```

Comment:
(See for `Optimizations <../admin/settings.rst#opendistro.query.size_limit>` details), the Optimizations should be Settings, right?

Reply:
yes, will change it